### ISSUE TITLE
Please read & provide the following

### ISSUE THREAD

**Issue by nuggy875** [facebookresearch/detr#505]
Do you have any performance issue when training with lower batch?
It performs well with batch size 16 but have performance issue with the batch lower than 8..

Does anyone have same problem with me?

I experimented with various GPU but same issue.

**Comment by Puranjay-del-Mishra** [comment]
I get an assertion error on my second training image, with the output bbox coordinates becoming NaN. Do you have faced anything like this?

**Comment by Fun772283153** [comment]
> Do you have any performance issue when training with lower batch? It performs well with batch size 16 but have performance issue with the batch lower than 8.. (training loss didn't seem to converge)
> 
> Does anyone have same problem with me?
> 
> I experimented with various GPU but same issue.

I have met the same problem, when using small batch size like 2, 4, after some iterations, the transformer's hidden state becomes the same, and ap is 0.
